charts/ingress-nginx/tests/controller-deployment_test.yaml

```
@@ -0,0 +1,33 @@
+suite: test controller deployment
+templates:
+  - controller-deployment.yaml
+
+tests:
+  - it: should always create a deployment
```

```suggestion
  - it: should create a Deployment
```

Always might be a bit misleading as `controller.kind: DaemonSet` does not create a Deployment.

---

And maybe because of this it's worth having a test for `controller.kind: DaemonSet` which checks for no documents.